WordPress/Classes/ViewRelated/Post/PostEditor+BlogPicker.swift

```diff
@@ -1,102 +0,0 @@
-import UIKit
-
-extension PostEditor where Self: UIViewController {
-
-    func blogPickerWasPressed() {
-        assert(isSingleSiteMode == false)
-        // if it's a reblog, we won't change the content, so don't show the alert
-        guard !postIsReblogged, post.hasSiteSpecificChanges()  else {
-            displayBlogSelector()
-            return
-        }
-
-        displaySwitchSiteAlert()
```

Comment:
Can we maybe remove some of these properties, methods as well? (`isSingleSiteMode, postIsReblogged, post.hasSiteSpecificChanges`, etc.)

Reply:
Removed `isSingleSiteMode` at 63d3a82 & `hasSiteSpecificChanges` at 0e996fe

I think We can't remove `postIsReblogged` as it's being used for _Site Switcher_ on Reader Section

Reply:
> I think We can't remove postIsReblogged as it's being used for Site Switcher on Reader Section

What do you think it might be used for in the Reader? 🤔

Reply:
I think for re-blogging a WP Blog

<img src="https://user-images.githubusercontent.com/16770566/115078769-bab19400-9f1d-11eb-9540-0821bdad3e0c.png" width=375>